challenges/20200602/billy/index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+    <head>
+        <title>
+            Vue Playground
+        </title>
+
+        <meta charset="UTF-8">
+        <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+    </head>
+
+    <body>
+        <div id="vueplayground">
+           <ul>
+               <li v-for="todo in list">
+                   {{todo.text}}
+               </li>
+               
```

Comment:
I noticed you have several blank lines between child elements and the end tag of the parent element in your code, specifically, between line 16-18, 22-24, 25-27. Please delete these lines.